opamp/helpers.go

```diff
@@ -0,0 +1,36 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opamp
+
+import (
+	"crypto/sha256"
+
+	"github.com/open-telemetry/opamp-go/protobufs"
+)
+
+// StringKeyValue converts a string key-value pair into a protobuf.KeyValue struct
+func StringKeyValue(key, value string) *protobufs.KeyValue {
+	return &protobufs.KeyValue{
+		Key: key,
+		Value: &protobufs.AnyValue{
+			Value: &protobufs.AnyValue_StringValue{StringValue: value},
+		},
+	}
+}
+
+// ComputeHash computes a sha256 hash of the passed in data
+func ComputeHash(data []byte) []byte {
+	return sha256.New().Sum(data)
+}
```

Comment:
Oh dang good call I'll fix that.

We use sha256 because that's the way opamp hashes things. Theres no reason I just figured we should be consistent. It's purely for comparing hashes.

Reply:
Cool, if it's for consistency that sounds like a good enough reason.

I ask because FNV is less expensive than SHA2, so I tend towards that for non-cryptographic purposes, but I don't think this is really a hot path, so it doesn't matter much here.

Reply:
Yeah this path involves writing and reading from disk so it's not fast at all